ConanInstall.cmake

```
@@ -59,28 +59,17 @@ if(NOT CONAN_OPENSTUDIO_ALREADY_RUN)
     ${CONAN_QT}
     openssl/1.1.0l # ruby 2.5.5 won't work with 1.1.1x, so use 1.1.0l here to try to force every package to align on the same as ruby
     # Track NREL/stable in general, on a feature branch this could be temporarily switched to NREL/testing
-    openstudio_ruby/2.5.5@nrel/stable    # TODO: Temp #@nrel/stable
-    boost/1.71.0
-    #boost_asio/${BOOST_VERSION}@bincrafters/stable
-    #boost_program_options/${BOOST_VERSION}@bincrafters/stable
-    #boost_regex/${BOOST_VERSION}@bincrafters/stable
-    #boost_filesystem/${BOOST_VERSION}@bincrafters/stable
-    #boost_crc/${BOOST_VERSION}@bincrafters/stable
-    #boost_algorithm/${BOOST_VERSION}@bincrafters/stable
-    #boost_uuid/${BOOST_VERSION}@bincrafters/stable
-    #boost_log/${BOOST_VERSION}@bincrafters/stable
-    #boost_numeric_ublas/${BOOST_VERSION}@bincrafters/stable
-    #boost_functional/${BOOST_VERSION}@bincrafters/stable
-    #boost_geometry/${BOOST_VERSION}@bincrafters/stable
-    pugixml/1.10@bincrafters/stable
-    jsoncpp/1.9.2
+    openstudio_ruby/2.5.5@nrel/testing    # TODO: Temp #@nrel/stable
```

```suggestion
    openstudio_ruby/2.5.5@nrel/stable    # TODO: Temp #@nrel/stable
```

Revert to stable now that NREL/conan-openstudio-ruby#28 was merged